src/tokens/selectors.test.ts

```diff
@@ -733,4 +734,61 @@ describe('swappableTokensByNetworkIdSelector', () => {
       },
     ])
   })
+
+  describe('importedTokensSelector', () => {
+    const mockState: any = {
+      tokens: {
+        tokenBalances: {
+          ...state.tokens.tokenBalances,
+          ['celo-alfajores:importedToken']: {
+            name: 'importedToken',
+            tokenId: 'celo-alfajores:importedToken',
+            networkId: NetworkId['celo-alfajores'],
+            balance: '0',
+            priceFetchedAt: mockDate,
+            minimumAppVersionToSwap: '1.10.0',
+            isManuallyImported: true,
+          },
+          ['ethereum-sepolia:importedToken']: {
+            name: 'importedToken',
+            tokenId: 'ethereum-sepolia:importedToken',
+            networkId: NetworkId['ethereum-sepolia'],
+            balance: '0',
+            priceFetchedAt: mockDate,
+            minimumAppVersionToSwap: '1.10.0',
+            isManuallyImported: true,
+          },
+        },
+      },
+    }
+
+    it('returns all the imported tokens', () => {
+      const importedTokens = importedTokensSelector(mockState, [
+        NetworkId['celo-alfajores'],
+        NetworkId['ethereum-sepolia'],
+      ])
+
+      expect(importedTokens).toMatchObject([
+        {
+          ...mockState.tokens.tokenBalances['celo-alfajores:importedToken'],
+          balance: new BigNumber(0),
```

Comment:
i wondered for a moment if this is a special assertion that the balance should always be 0 when returned from the imported tokens selector. then i realised the assertion is on the big number type. perhaps we can add a non zero balance for one of these tokens?